test/functional/wallet_blank.py

```
@@ -0,0 +1,170 @@
+)
+from test_framework.wallet_util import bytes_to_wif
+
+
+class WalletBlankTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def add_options(self, options):
```

Ruff (0.12.2)

Missing return type annotation for public function `add_options`

Add return type annotation: `None`

(ANN201)